code/modules/mob/living/carbon/human/species/outsider/revenant.dm

```diff
@@ -34,18 +34,15 @@
 	ethanol_resistance = -1
 	taste_sensitivity = TASTE_NUMB
 
-	speech_sounds = list('sound/species/revenant/grue_growl.ogg')
```

Comment:
Remove this file if it's no longer used.

Reply:
Unneeded, it's still used for a revenant ability, which is on a cooldown and isn't as spammy as being attached to a 50/50 sound effect when talking in-character.